src/pages/resources/recipes/flyout-menu/index.md

```diff
@@ -0,0 +1,70 @@
+---
+keywords:
```

Comment:
The keywords should be relevant to the doc making it easier for the devs to find the topic. Can you update this list?

Reply:
Comment Addressed.